files/preStudies/frameworks.tex

```diff
@@ -0,0 +1,80 @@
+\section{Frameworks}
+
+This section will evaluate different frameworks for cross-platform development which could be used in this project. Some must have and nice to have capabilities will be
+	
+\textbf{Must have:}
+	\begin{itemize}
+	\item Cross platform capabilities
+	\item Wide community support
+	\item \textit{Bluetooth Low Energy} capabilities
+	\end{itemize}
+	
+\textbf{Nice to have:}
+	
+	\begin{itemize}
+	\item 1 platform to rule them all
+	\item type safety
+	\item reactive data
+	\item compiles to native code for increased performance
+	\end{itemize}
+	
```

Comment:
Maybe a short explanation of the nice to haves. What is meant by 1 platform and reactive data.